CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+### 1.20.2
+
+* Raise error from Cucumber Queue Mode runner when system process doesn't finish execution
```

Comment:
Let's change this to a more explicit example:

Raise an error when running Cucumber in Queue Mode and Cucumber system process doesn't finish execution correctly (for instance Cucumber process was killed by CI server due to lack of memory)